packages/mui-base/src/useSelect/useSelect.types.ts

```diff
@@ -93,6 +103,14 @@ export interface UseSelectParameters<OptionValue, Multiple extends boolean = fal
    * If this parameter is set, options defined as JSX children are ignored.
    */
   options?: SelectOptionDefinition<OptionValue>[];
+  /**
+   * A function to convert the currently selected value to a string.
+   * Used to set a value of a hidden input associated with the select,
+   * so that the selected value can be posted with a form.
+   */
+  getSerializedValue?: (
+    option: SelectValue<SelectOption<OptionValue>, Multiple>,
+  ) => React.InputHTMLAttributes<HTMLInputElement>['value'];
```

Comment:
```suggestion
  ) => string;
```

As you suggested, it should be a string to be sensibly serializable (same in Select.types.ts).

Reply:
Done, thanks!